### ISSUE TITLE
Is batch normalization still used in training mode when sampling examples (testing) ?

### ISSUE THREAD

**Issue by watsonyanghx** [duxingren14/DualGAN#10]
I see you don't specify the running mode in [batch_norm](https://github.com/duxingren14/DualGAN/blob/master/ops.py#L8). Is there any reasons? Does this have a big influence in the quality of image generated?

**Comment by duxingren14** [comment]
Thanks for pointing out this. Actually I used the default setting of BatchNorm for all my experiments. No further trials indicate if a different running model does bring any influence. Anyway it is an interesting topic that deserves a trial.

**Comment by watsonyanghx** [comment]
Thanks for your reply.

I have tried to split `train` and `test` phase, and the images sampled in **training mode** looks good, but when changed to **testing mode** (set `is_training=False` of batch normalization), all the images generated looks very bad， they are just noisy points.
